### PR TITLE
refactor(swap): store original swap input and remove old handlers

### PR DESCRIPTION
**Pull request by acdibble**

### CODE REVIEW

File: packages/swap/prisma/migrations/20240919124938_add_swap_input_amount/migration.sql
Comment (acdibble) [inline packages/swap/prisma/migrations/20240919124938_add_swap_input_amount/migration.sql:1]: important

File: packages/swap/src/event-handlers/swapRequested.ts [hunk 1]
```diff
@@ -121,6 +121,7 @@ export default async function swapRequested({
       srcAddress: ccmMetadata?.sourceAddress?.address,
       destAddress,
       swapRequestedAt: new Date(block.timestamp),
+      swapInputAmount: inputAmount.toString(),
```
Comment (acdibble) [inline packages/swap/src/event-handlers/swapRequested.ts:124]: important

File: packages/swap/src/event-handlers/ccmDepositReceived.ts [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-export default async function ccmDepositReceived({ prisma, event, block }: EventHandlerArgs) {
-  const { principalSwapId, depositMetadata } = ccmDepositReceivedArgs.parse(event.args);
-
-  if (principalSwapId) {
-    await prisma.swapRequest.update({
-      where: {
-        nativeId: principalSwapId,
-      },
-      data: {
-        ccmDepositReceivedBlockIndex: `${block.height}-${event.indexInBlock}`,
```
Comment (niklasnatter) [inline packages/swap/src/event-handlers/ccmDepositReceived.ts:32]: can we remove this field? thinks its not used anymore

**Review by niklasnatter — APPROVED** [review]

**Review by jerryafr — APPROVED** [review]

File: packages/swap/src/event-handlers/ccmDepositReceived.ts (hunk 2 above)
Comment (acdibble) [reply packages/swap/src/event-handlers/ccmDepositReceived.ts:32]: It's used in the status endpoint, we'd have to deprecated it first but we are already deprecating that endpoint anyway

**Pull request merged**